api/python/cell_census/src/cell_census/get_anndata.py

```diff
@@ -37,9 +37,17 @@ def get_anndata(
     obs_value_filter: str, default None
         Value filter for the ``obs`` metadata. Value is a filter query written in the
         SOMA ``value_filter`` syntax.
+    obs_coords: Tuple[SparseDFCoord, ...]
+        A set of coordinates on the ``obs`` DataFrame index. The tuple must have a length less 
+        than or equal to the number of dimensions, and be of a type supported by 
+        SOMA ``DataFrame``.
```

Comment:
This is a larger question than just the docs, but shouldn't `get_anndata()` just take a single `SparseDFCoord` rather than a tuple? Since cell census only has 1 dimension, it seems like any CC API method that takes coords should hide this multi-dimension capability of SOMA.

Reply:
I agree with you and I'm trying to thread the line between improved UX for comp bios and consistency with tiledbsoma. This will be a continued conversation we'll have for many functionalities of the census -- API related to #78. I'm leaning towards your suggestion for this case.

Do you have any other extra thoughts on this @atolopko-czi ?

Reply:
I think we just have to take each design choice on a case-by-case basis (i.e. "continued conversation"). If we _always_ try to maintain consistency with tiledbsoma APIs, we might restrict our design choices too much.

Reply:
Tracking here #119